### PR TITLE
CbG Experiment + Bicluster Data LFs

### PR DESCRIPTION
**Pull request by danich1**
This PR tested the hypothesis that DaG label functions will transfer onto CbG sentences.  Notebook 3 has results for this experiment. As one can see DaG lf doesn't add towards the performance. The next experiment will be seeing how well both DaG and CbG functions work on Chemical Treats Disease relationship.

### CODE REVIEW

**Review by dhimmel — COMMENTED** [review]
I don't love that the DaG notebooks are being overwritten. Ideally we would have separate notebooks or cells or loops to accommodate different edge types. If you find code is being repeated, then perhaps that code should be put into modules.

I'm not going to say that this PR shouldn't proceed overwriting DaG work, but that you should think about modularity and supporting multiple relationship types simultaneously going forward. This can be difficult with noteobooks, so you may have to modularize some code into .py files. 

> The next experiment will be seeing how well both DaG and CbG functions work on Chemical Treats Disease relationship.

Regarding CbG labeling functions, I'd recommend one per source as we do for DaG Hetionet-derived LFs.

This will be my last review before my travels. So while I'm gone I don't think it makes sense to wait on my approval for PRs. Proceed as you see fit.

**Comment by danich1** [comment]
Great Point. I made changes to reflect this suggestion. Since @dhimmel is out of town, I am adding @ajlee21 to review this PR.

File: All_Relationships/2.candidate-labeler.py
Comment (ajlee21) [inline All_Relationships/2.candidate-labeler.py:151]: I would suggest adding comments for your different functions so that when you return to these scripts later you remember what they were input/output is and what they were used for

Comment (ajlee21) [reply All_Relationships/2.candidate-labeler.py:151]: Same for get_split that is above

File: All_Relationships/utils/datafiles/compound_gene_datafiles.py
Comment (ajlee21) [inline All_Relationships/utils/datafiles/compound_gene_datafiles.py]: Minor cleanup - I would create a filename variable that you can easily change in the beginning of the code.  Not sure if this file will always be stored/named the same because in that case perhaps you don't want the user to accidentally have access and then change it

File: All_Relationships/utils/disease_gene_lf.py
Comment (ajlee21) [inline All_Relationships/utils/disease_gene_lf.py:101]: I would try to be more consistent about having comments for what each of the label functions are doing and also what the category is too

File: All_Relationships/3.model-label-functions.py
Comment (ajlee21) [inline All_Relationships/3.model-label-functions.py:151]: same comment as above

File: All_Relationships/3.model-label-functions.py
Comment (ajlee21) [inline All_Relationships/3.model-label-functions.py]: I might not understand the meaning of the plots here when we went over them together so I don't know if you want to add comments here/change the title of the axes

File: All_Relationships/3.model-label-functions.py
Comment (ajlee21) [inline All_Relationships/3.model-label-functions.py:421]: Why do you have this block commented out?

**Review by ajlee21 — APPROVED** [review]
Basically just a bunch of minor edits: adding comments to give clarity

File: All_Relationships/3.model-label-functions.py
Comment (danich1) [reply All_Relationships/3.model-label-functions.py:421]: That is blocked out because I didn't want to run that section of the notebook for this experiment. Granted I'm sure there is a better way to handle this situation, but it sufficed for now.

Comment (cgreene) [reply All_Relationships/3.model-label-functions.py:421]: This is lightly worrisome and, like @dhimmel, makes me wonder if some of the code in these notebooks should really be functions that are imported and called.

Comment (danich1) [reply All_Relationships/3.model-label-functions.py:421]: Ah ok. Don't want to blow up this PR with a lot of changes, so I am going to upload the changes I made (per the above comments) and then open a new PR for some code restructure and housekeeping.

**Pull request merged**
